task/common/cloud.go

```diff
@@ -29,6 +30,34 @@ const (
 	ProviderK8S Provider = "k8s"
 )
 
+type Credentials struct {
+	*AWSCredentials
```

Comment:
Let's not do this, we gain nothing from using anonymous struct members here.

Reply:
Resolved with https://github.com/iterative/terraform-provider-iterative/pull/710/commits/d95f7d8762972d0ad98ac581b1679909e8d59eb1 😄